### PR TITLE
Build fails due to a dependency on bintray

### LINKED ISSUE #1864: Build fails due to a dependency on bintray
# Summary

As of May 1st, 2021, Bintray is no longer in service. Refer [this](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/) article. Since one of the libraries in the project depend on Bintray (refer [this](https://github.com/openMF/android-client/blob/15616d3dd507b17050e9c933303c104cd4d9ddc2/mifosng-android/build.gradle#L31)) the build is throwing the following error
```bash
Could not GET 'http://dl.bintray.com/amulyakhare/maven/androidx/navigation/navigation-runtime/maven-metadata.xml'. Received status code 502 from server: Bad Gateway
Disable Gradle 'offline mode' and sync project
```

# Steps to reproduce

Open the project in the latest version of Android Studio and build the application.

# Expected behavior

After cloning, the app should build successfully without any issues

# Screenshots

Screenshot 2022-03-18 at 10 01 40

# Resolution

Some contributors have forked and published the library to a different distribution service. Refer [this](https://github.com/amulyakhare/TextDrawable/issues/80).
Import the library from any such source

### PR DESCRIPTION
**Pull request by avinash14022002**
Fixes #1864 , #1865

Please Add Screenshots If there are any UI changes.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: mifosng-android/build.gradle [hunk 1]
```diff
@@ -147,6 +146,7 @@ tasks.withType(JavaCompile) {
 
 dependencies {
     implementation 'androidx.multidex:multidex:2.0.0'
+    implementation 'com.amulyakhare:com.amulyakhare.textdrawable:1.0.1'
```
Comment (danishjamal104) [inline mifosng-android/build.gradle:149]: where are we using this library?

File: mifosng-android/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -10,7 +10,7 @@
         <!-- Customize your theme here. -->
     </style>
 
-    <style name="MaterialAppTheme" parent="Theme.AppCompat.DayNight.NoActionBar">
+    <style name="MaterialAppTheme" parent="Theme.MaterialComponents.DayNight.NoActionBar">
```
Comment (danishjamal104) [inline mifosng-android/src/main/res/values/styles.xml:13]: have you tested this on multiple android versions?

**Review by danishjamal104 — CHANGES_REQUESTED** [review]
Can you confirm that this PR is fixing the proposed issues by adding screenshot and short video in PR template

File: mifosng-android/build.gradle (hunk 1 above)
Comment (avinash14022002) [reply mifosng-android/build.gradle:149]: The dependencies was used in [maven { url 'http://dl.bintray.com/amulyakhare/maven' }](https://github.com/openMF/android-client/pull/1866/files#diff-f60af60b2e4a2537c2402faacea0730fc3f194c4359324498f39088f4cea0692L31) which generates error on compilation.

File: mifosng-android/src/main/res/values/styles.xml (hunk 2 above)
Comment (avinash14022002) [reply mifosng-android/src/main/res/values/styles.xml:13]: No,
how to check that?
by downloading different version android on android studio?

**Comment by avinash14022002** [comment]
> Can you confirm that this PR is fixing the proposed issues by adding screenshot and short video in PR template

@danishjamal104 
image

I was trying to upload short video, I guess currently server are down unable to login

File: mifosng-android/src/main/res/values/styles.xml (hunk 2 above)
Comment (danishjamal104) [reply mifosng-android/src/main/res/values/styles.xml:13]: create virtual devices with multiple dpis. [Checkout this](https://developer.android.com/studio/run/managing-avds)

**Comment by avinash14022002** [comment]
@danishjamal104 @iamsh4shank ,
The code is reviewed by @danishjamal104 ,
possible to be merge

**Comment by iamsh4shank** [comment]
Done @avinash14022002

**Pull request merged**